cmd/template/main.go

```diff
@@ -48,3 +48,33 @@ clean:
 .PHONY: all build run test clean
 		`)
 }
+
+func ReadmeTemplate() []byte {
+	return []byte(
+		`
+# Project Title
+
+One Paragraph of project description goes here
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
+
+## MakeFile
+
+run all make commands with clean tests
+make all build
+
+build the application
+make build
+
+run the application
+make run
+
+run the test suite
+make test
+
+clean up binary from the last build
+make clean
```

Comment:
Should we consider putting these commands in a code block?

Reply:
i tried to figure that one out.... you see the issue is the ` character. the template literal string thinks its being delaminated due to markdown using 

```
code block
```

new to go so was not sure how to fix.

Reply:
This is a possible workaround:

````
`
## MakeFile

run all make commands with clean tests
` + "```bash" + `
make all build
` + "```" + `

build the application

` + "```bash" + `
make build
` + "```" + `
`
````

Reply:
see latest commits